.travis/config.hpp

```diff
@@ -10,45 +10,54 @@
 // of the License or (at your option) any later version.
 // (see http://www.opensource.org/licenses for more info)
 
-
-
 #if !defined(ARMA_USE_LAPACK)
```

Comment:
We should irgnore this file too; you can take a look at the existing linter script which file is excluded from the check: https://github.com/mlpack/jenkins-conf/blob/master/linter/lint.sh